includes/class-sensei-updates.php

```diff
@@ -198,6 +199,13 @@ public function __construct( $parent ) {
 
 	} // End __construct()
 
+	/**
+	 * Initializes the updates that have been executed.
+	 */
+	public function init_updates_run() {
+		$this->updates_run = self::get_ran_upgrades_raw();
```

Comment:
We could do additional page checks in WP admin, but this class is about to be deprecated.